### PR TITLE
security(snyk): removed high level snyk issues for hard coded credentials

### PR DESCRIPTION
**Pull request by pacificcode**
removed high level snyk issues for hard coded credentials

### CODE REVIEW

File: constants/commands.go [hunk 1]
```diff
@@ -2,30 +2,29 @@ package constants
...
-	Create         = "create"
-	Update         = "update"
-	Rollback       = "rollback"
-	Upload         = "upload"
-	Edit           = "edit"
-	Delete         = "delete"
-	Describe       = "describe"
-	Clear          = "clear"
-	List           = "list"
-	ChangePassword = "change-password"
```
Comment (sheldonhull) [inline constants/commands.go:15]: Was this meant to be removed? If so and it impacts usage make sure to include a changie entry for each change type.

Comment (sheldonhull) [reply constants/commands.go:15]: @andrii-zakurenyi  can you put a second review on this so I'm not missing something?

File: .changes/unreleased/security-20240812-133345.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+kind: security
+body: removed high level snyk issues for hard coded credentials
```
Comment (sheldonhull) [inline .changes/unreleased/security-20240812-133345.yaml]: Please fix so it's a sentence with punctuation. Will be a bullet point in changelog

**Review by sheldonhull — CHANGES_REQUESTED** [review]

File: constants/commands.go (hunk 1 above)
Comment (pacificcode) [reply constants/commands.go:15]: Full file is here, https://github.com/DelineaXPM/dsv-cli/blob/9fc8fe91bbb8639978a2973bcf174d28f2e01483/constants/commands.go
Only the one variable was removed. This does not impact usage.

File: .changes/unreleased/security-20240812-133345.yaml (hunk 2 above)
Comment (pacificcode) [reply .changes/unreleased/security-20240812-133345.yaml]: done :)

**Review by andrii-zakurenyi — APPROVED** [review]
Good for snyk, bad for the codebase, but I get it and have no ideas on how to resolve it in a better way :)

**Comment by pacificcode** [comment]
@andrii-zakurenyi I totally agree.

**Review by sheldonhull — APPROVED** [review]

**Pull request merged**